assets/src/components/account/GettingStartedOverview.tsx

```diff
@@ -317,6 +331,21 @@ class GettingStartedOverview extends React.Component<Props, State> {
             />
           </Box>
 
+          <Box mb={3}>
+            <label htmlFor="greeting">
+              Set an away message (will replace greeting message outside working
+              hours):
+            </label>
+            <Input
+              id="greeting"
+              type="text"
+              placeholder="We're not at the office right now!"
```

Comment:
haha feels weird to imagine being in an office during covid times 😛 

maybe let's just have the placeholder be "Sorry, we're away at the moment!"